config/seata-config-apollo/src/main/java/io/seata/config/apollo/ApolloConfiguration.java

```diff
@@ -176,6 +180,12 @@ private void readyApolloConfig() {
                 System.setProperty(PROP_APOLLO_SECRET, secretKey);
             }
         }
+        if (!properties.containsKey(APOLLO_CLUSTER)) {
+            System.setProperty(PROP_APOLLO_CLUSTER, FILE_CONFIG.getConfig(getApolloCluster()));
+        }
+        if (!properties.containsKey(APOLLO_CONFIG_SERVICE)) {
+            System.setProperty(PROP_APOLLO_CONFIG_SERVICE, FILE_CONFIG.getConfig(getApolloConfigService()));
```

Comment:
If the value is ""?

Reply:
It is empty.Apollo will use apolloMeta.If apolloMeta is empty.The addr of apollo will be null.